types/react-big-calendar/tsconfig.json

```diff
@@ -20,6 +20,7 @@
     },
     "files": [
         "index.d.ts",
-        "react-big-calendar-tests.tsx"
+        "react-big-calendar-tests.tsx",
+        "lib/addons/dragAndDrop.d.ts"
     ]
-}
\ No newline at end of file
+}
```

Comment:
nit: reintroduce trailing newline

Reply:
It's there, I don;t know what is up with the GitHub diff view. 🤷‍♂️